src/ethernet/ip_device.hh

```diff
@@ -0,0 +1,82 @@
+#pragma once
+
+//////////////////////////////////////////////////////////////////////////
+#ifdef _WIN32
+
+#define _WINSOCKAPI_
+
+#include <windows.h>
+#pragma comment(lib, "Ws2_32.lib")
+
+#include <SIGNAL.H>
+
+#endif
+//////////////////////////////////////////////////////////////////////////
+
+#include <librealsense2/rs.hpp>
+#include <librealsense2/hpp/rs_internal.hpp>
+#include "option.h"
+
+#include "RsRtspClient.h"
+#include "software-device.h"
+
+#include <list>
+
+#include "rs_rtp_stream.hh"
+#include "rs_rtp_callback.hh"
+#include "ip_sensor.hh"
+
+#define MAX_ACTIVE_STREAMS 4
+
+#define NUM_OF_SENSORS 2
+
+#define POLLING_SW_DEVICE_STATE_INTERVAL 100
+
+class ip_device
+{
+
+public:
+#ifdef _WIN32
+    __declspec(dllexport)
+#endif
+    
+    ip_device(std::string ip_address, rs2::software_device sw_device);
+
+    ~ip_device();
+
+    ip_sensor *remote_sensors[NUM_OF_SENSORS];
```

Comment:
Please use `std::vector<unique_ptr<ip_sensor>>` instead, it's better

Reply:
make_shared is not fully supported at c++11. as well at my setup.
@dorodnic , are you suggesting to use shared_ptr or to leave it as is for now? 
my source: 
[](https://stackoverflow.com/questions/24609271/errormake-unique-is-not-a-member-of-std)

Reply:
after viewing the ip_sensors usages I will use shared_ptr.  WIP.

Reply:
Not blocking issue. 
New task for post release was created: https://rsjira.intel.com/browse/CAMOE-279